client/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hp-admin-key-manager"
```

Comment:
I'm in favor of calling this `hp-admin-keypair` if everyone else likes that better

Reply:
done